assets/js/stripe-payment-request.js

```diff
@@ -147,37 +147,47 @@ jQuery( function( $ ) {
 		},
 
 		/**
-		 * Abort payment and display error messages.
+		 * Display error messages.
 		 *
-		 * @since 3.1.0
-		 * @version 4.0.0
-		 * @param {PaymentResponse} payment Payment response instance.
-		 * @param {String}          message Error message to display.
+		 * @since 4.8.0
+		 * @param {Object} message DOM object with error message to display.
 		 */
-		abortPayment: function( payment, message ) {
-			payment.complete( 'fail' );
-
+		displayErrorMessage: function( message ) {
 			$( '.woocommerce-error' ).remove();
 
 			if ( wc_stripe_payment_request_params.is_product_page ) {
-				var element = $( '.product' );
-
+				var element = $( '.product' ).first();
 				element.before( message );
 
 				$( 'html, body' ).animate({
 					scrollTop: element.prev( '.woocommerce-error' ).offset().top
 				}, 600 );
 			} else {
 				var $form = $( '.shop_table.cart' ).closest( 'form' );
+				// Need to check if element exists, otherwise an error is thrown on checkout.
```

Comment:
👍 – did you run into this / is it easily reproducible?

Reply:
Yes, I ran into this. An error is thrown at checkout because the element doesn't exist. 

If I remember it correctly, the function was made to work only on the product page and cart page, but the startPayment is triggered on the checkout page (maybe something to be improved), and would call this function, causing the error.